h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -229,7 +232,7 @@ public AutoMLBuildSpec getBuildSpec() {
 
   private Date startTime;
   private static Date lastStartTime; // protect against two runs with the same second in the timestamp; be careful about races
-  private long stopTimeMs;
+  private Countdown runCountdown;
```

Comment:
@seb-h2o runCountdown is slightly confusing as we have two types of countdowns... in terms of number of models and in terms of time that is left. wdyt?

Reply:
@deil87 talking about the name? countdown is usually about time, isn't it?

Reply:
@seb-h2o yes, about naming. Probably you are right. Even thought definition is " the act of counting backwards to zero" I found only examples that are related to time. I was confused because of the CountDownLatch class in Java.

Reply:
strictly speaking, could be renamed to `CountdownTimer`, but can't do this here as it requires changes in core module :)

Reply:
@seb-h2o ahah :)